app/2.0/docs/devguide/dom-template.md

```diff
@@ -83,4 +155,138 @@ use the standard DOM `querySelector`  method.
 
 
 
+## Remove empty text nodes {#strip-whitespace}
+
+
+Add the `strip-whitespace` boolean attribute to a template to remove
+any empty text nodes from the template's contents. This can result in a
+minor performance improvement.
```

Comment:
We recently had a user confused that `strip-whitespace` wasn't actually stripping whitespace out of the _inside_ of text nodes... it only strips whitespace-only text nodes between other elements.  Might be worth mentioning?